### PR TITLE
Feature/add forms to content media

### PR DESCRIPTION
**Pull request by mattwendzina**
## 📝 Description

Adds Forms into Content Media

## ⛳️ Current behavior (updates)

Currently Content Media only allows Images

## 🚀 New behavior

This allows Forms to be added to Content Media

## 💣 Is this a breaking change (Yes/No):

No

### CODE REVIEW

File: packages/osc-ecommerce/app/components/ContentMedia/ContentMedia.tsx [hunk 1]
```diff
@@ -140,47 +141,42 @@ const ContentMediaBlockModule = (props: ContentMediaBlockProps) => {
...
+                            spacing: 16,
                         },
-                    }}
-                >
-                    {media?.mediaType.map((media) =>
-                        media?.image ? (
+                    },
+                }}
+            >
+                {media?.mediaType.map((media) => {
```
Comment (stevenbister) [inline packages/osc-ecommerce/app/components/ContentMedia/ContentMedia.tsx:177]: Getting an eslint error on this line, doesn't look like we're returning anything in some cases

File: packages/osc-ecommerce/app/components/Forms/Forms.tsx [hunk 2]
```diff
@@ -93,9 +93,9 @@ const Form = (props: FormProps) => {
     );
 };
 
-export const Forms = (props: { module: formModule }) => {
+export const Forms = (props: { module: formModule; addContainer?: boolean }) => {
     // Module data coming from Sanity
-    const { module } = props;
+    const { module, addContainer = false } = props;
```
Comment (stevenbister) [inline packages/osc-ecommerce/app/components/Forms/Forms.tsx:98]: Is there a reason for using this prop over adding the container class directly like the other components?

**Review by stevenbister — CHANGES_REQUESTED** [review]
Nice man, one eslint warning, and a question from me :)

Do you also think it's worth adjusting the space between the two columns when the form is set? They feel a bit close at the moment 🤔

File: packages/osc-ecommerce/app/components/Forms/Forms.tsx (hunk 2 above)
Comment (mattwendzina) [reply packages/osc-ecommerce/app/components/Forms/Forms.tsx:98]: As discussed, some cases, like ContentMedia won't want a container.

**Comment by mattwendzina** [comment]
> Nice man, one eslint warning, and a question from me :)
> 
> Do you also think it's worth adjusting the space between the two columns when the form is set? They feel a bit close at the moment 🤔

I found a way to add this conditionally in `ContentMedia.tsx` and have added a 40/60 split to the script for whenever ContentMedia has a form. I believe this creates the level of space we discussed. See screenshot below:

Screenshot 2023-05-15 at 15 04 32

**Comment by stevenbister** [comment]
> I found a way to add this conditionally in `ContentMedia.tsx` and have added a 40/60 split to the script for whenever ContentMedia has a form. I believe this creates the level of space we discussed. See screenshot below:
> 
> Screenshot 2023-05-15 at 15 04 32

Sorry mate, not sure if this is quite working for me? The `start-7` bit is there but the other column is still the same width? Do I need to do something to set it? 

Also; if we swap the order, so form on the left and content on the right then it goes a bit wonky
image

These are the settings for the slide:
image

**Review by stevenbister — APPROVED** [review]
LGTM :)

**Pull request merged**